manifests/m/Microsoft/SQLServerManagementStudio/19.0.1/Microsoft.SQLServerManagementStudio.installer.yaml

```diff
@@ -0,0 +1,138 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: '19.0.1'
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+FileExtensions:
+- SQL
+ReleaseDate: 2023-02-02
+Installers:
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: burn
+  Scope: machine
+  InstallerUrl: https://go.microsoft.com/fwlink/?linkid=2223741&clcid=0x407
+  InstallerSha256: A8EDFE4BA29C8F307CD6D5FB3FED650D23EE46A6179DE86EA7337B2434A76CCE
+  AppsAndFeaturesEntries:
+  - DisplayName: Microsoft SQL Server Management Studio - 19.0.1
+    DisplayVersion: 
+    ProductCode: '19.0.20200.0'
```

Comment:
```suggestion
```
This value is incorrectly put under `ProductCode` and should be under `DisplayVersion`. It's the same case for every installer node in the manifest.

Reply:
Nice catch. Let me fix the script that generated it. Thanks.